src/SFA.DAS.EmployerUsers.Application/Commands/ActivateUser/ActivateUserCommandValidator.cs

```diff
@@ -8,6 +8,9 @@ namespace SFA.DAS.EmployerUsers.Application.Commands.ActivateUser
 {
     public class ActivateUserCommandValidator : IValidator<ActivateUserCommand>
     {
+        private string _invalidCodeMessage = "Code is not recognised";
+        private string _missingCodeMessage = "Enter the code we emailed you";

```

Comment:
add a const keyword and update the name as appropriate

Reply:
As per comment on other; changed.